src/sentry/data_export/endpoints/data_export.py

```diff
@@ -79,7 +79,13 @@ def validate(self, data):
                 organization_id=organization.id,
             )
             try:
-                get_filter(query_info["query"], processor.params)
+                snuba_filter = get_filter(query_info["query"], processor.params)
+                resolve_field_list(
+                    fields,
```

Comment:
Might need to do a shallow copy on `fields`. Otherwise `resolve_field_list()` may mutate the list. i.e. convert `project.id` to `project`.

```suggestion
                    fields.copy(),
```

Reply:
Good point